index.js

```diff
@@ -0,0 +1,149 @@
+const compareLangKeys = () => {
+  // lib is needed to show deuplicated keys
+  const jsonValidator = require('json-dup-key-validator');
+
+  //requiring path and fs modules
+  const path = require('path');
+  const fs = require('fs');
+  const directoryPath = path.join(__dirname, 'locales');
+
+  const mapAllFiles = (files, filesContent) => {
+    const isJsonFile = (JSONdata, file) => {
+      const singleFilesContent = [];
+
+      for (const key in JSONdata) {
+        singleFilesContent.push(key);
+      }
+      filesContent.push({
+        fileName: file,
+        fileKeys: singleFilesContent
+      });
+    };
+
+    const isNotJsonFile = () => {
+      notJSON++;
+      filesContent.push({
+        fileName: 'null',
+        fileKeys: []
+      });
+    };
+
+    const loadFiles = (file, isJsonFile, isNotJsonFile) => {
+      // load file if it is a JSON
+      if (file.includes('.json')) {
+        const JSONdata = require(`./locales/${file}`);
```

Comment:
Consider making the root folder configurable either through a parameter, or process current working directory `cwd`.